mods/mapgen/biomes.lua

```diff
@@ -1,398 +1,166 @@
+		local alpine    = "nodes_nature:highland_soil"
+		local basalt    = "nodes_nature:basalt"
+		local brick_gr  = "nodes_nature:granite_brick"
+		local brick_ls  = "nodes_nature:limestone_brick"
+		local clay      = "nodes_nature:clay"
+		local clay_wet  = "nodes_nature:clay_wet"
+		local drystack  = "tech:drystack"
+		local duff      = "nodes_nature:woodland_soil"
+		local duff_dry  = "nodes_nature:woodland_dry_soil"
+		local duff_wet  = "nodes_nature:woodland_soil_wet"
+		local dune      = "nodes_nature:duneland_soil"
+		local gneiss    = "nodes_nature:gneiss"
+		local granite   = "nodes_nature:granite"
+		local grass_bar = "nodes_nature:grassland_barren_soil"
+		local grass_wet = "nodes_nature:grassland_soil_wet"
+		local gravel    = "nodes_nature:gravel"
+		local gravel_w  = "nodes_nature:gravel_wet"
+		local gravel_ws = "nodes_nature:gravel_wet_salty"
+		local ice       = "nodes_nature:ice"
+		local lava      = "nodes_nature:lava_source"
+		local limestone = "nodes_nature:limestone"
+		local loam      = "nodes_nature:loam"
+		local marsh_wet = "nodes_nature:marshland_soil_wet"
+		local potable   = "nodes_nature:freshwater_source"
+		local prairie   = "nodes_nature:grassland_soil"
+		local sand      = "nodes_nature:sand"
+		local sand_wet  = "nodes_nature:sand_wet"
+		local sand_ws   = "nodes_nature:sand_wet_salty"
+		local silt      = "nodes_nature:silt"
+		local silt_wet  = "nodes_nature:silt_wet"
+		local silt_ws   = "nodes_nature:silt_wet_salty"
+		local snow      = "nodes_nature:snow"
+		local snow_b    = "nodes_nature:snow_block"
+		local stair_ds  = "stairs:stair_drystack"
+		local stair_gr  = "stairs:stair_granite_block"
+		local stair_ls  = "stairs:stair_limestone_block"
+--[[ Biome Descriptions ]]--[[
+	01.	Grassland: clay, open, yellow
+	02.	Upland Grassland
+	03.	Marshland: silt, dense reeds, red
+	04.	Highland: gravel, dense grasses, purple
+	05.	Duneland: sand, barren, orange
+	06.	Woodland: loam, trees, green
+	07.	Mountain Snowcaps: frozen places up high
+	08.	Coasts and Oceans: Silty Beach
+	09.	Lower Silty Beach
+	10.	Sandy Beach
+	11.	Lower Sandy Beach
+	12.	Gravel Beach
+	13.	Lower Gravel Beach
+	14.	Shallow Ocean
+	15.	Deep Ocean
+	16.	Underground
+	17.	Deep Underground
+	18.	Dry Grassland: clay, open, yellow, drier version of standard grassland
+	19.	Wet Grassland: Wet version of standard grassland
+	20.	Barren Grassland: pale colours
+	21.	Upland Dry Grassland
+	22.	Upland Barren Grassland
+	23.	Barrenland: gravel, lifeless
+	24.	Lavaland: cooled basalt and lava flows, totally lifeless
+	25.	Hardpan Marshland: thin silt, dense reeds, red, clay layer underneath
+	26.	Wet Woodland: loam, trees, green, over a layer of wet clay
+	27.	Dry Woodland: arid forest
+	28. Upland Woodland Dry
+	29.	Highland Scree: gravel, unstable loose gravel fields
+	30. Wet Highland Scree
+	31.	Dry Highland: no snow version
+	32.	Dry Nountain: bare rock, no snow
+  ]]
+-- 16. `underground` biome used to have basalt dungeon
```

Comment:
I moved the comments out of the data-table and made them use more-informative phrasing.